finetune/base_models/bert/featurizer.py

```diff
@@ -329,23 +350,22 @@ def chunk_aggregation(pooled_outputs):
                 pool_shape[2],
             ],
         )
-        pooled_output = tf.compat.v1.Print(
-            pooled_output, ["pooled_rs", tf.shape(pooled_output)], summarize=-1
-        )
+        # pooled_output = tf.compat.v1.Print(
+        #     pooled_output, ["pooled_rs", tf.shape(pooled_output)], summarize=-1
+        # )
 
         # Reduce across chunk dim with aggregation operation
-        # TODO implement aggregation options other than mean and attention
         features = chunk_aggregation(pooled_output)
-        features = tf.compat.v1.Print(
-            features, ["features", tf.shape(features)], summarize=-1
-        )
+        # features = tf.compat.v1.Print(
+        #     features, ["features", tf.shape(features)], summarize=-1
+        # )
 
         # FIXME In original featurizer, this reshape appears not to change dims [batch_size, hidden_size],
         # but without it there are shape errors in the target model
-        features = tf.reshape(
-            features,
-            shape=tf.concat((initial_shape[:-1], [config.n_embed]), 0),
-        )
+        # features = tf.reshape(
```

Comment:
We need this - but I'm pretty sure you've removed it to keep the final 1 or 2 dim in the features.